.env.example

```
@@ -37,4 +37,17 @@ NEXT_PUBLIC_SUPABASE_ANON_KEY=
+# AI Services (optional - AI features skip if not set)
+ANTHROPIC_API_KEY=
+
+# Email Services (optional - email features skip if not set)
+RESEND_API_KEY=
+
+# Vercel Deployment (automatically set by Vercel, no manual configuration needed)
+# VERCEL=1
+# VERCEL_ENV=preview|production|development
+# VERCEL_URL=<auto-generated-url>
+# VERCEL_GIT_COMMIT_SHA=<commit-sha>
+# VERCEL_GIT_COMMIT_REF=<branch-name>
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)